package.json

```diff
@@ -1,40 +1,7 @@
 {
-  "name": "@metamask/eslint-config",
-  "version": "5.0.0",
-  "description": "Shareable MetaMask ESLint config",
-  "main": "config/index.js",
-  "publishConfig": {
-    "registry": "https://registry.npmjs.org/",
-    "access": "public"
-  },
-  "files": [
-    "config/"
-  ],
-  "scripts": {
-    "test": "eslint ."
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/MetaMask/eslint-config.git"
-  },
-  "author": "Whymarrh Whitby <whymarrh.whitby@gmail.com>",
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/MetaMask/eslint-config/issues"
-  },
-  "homepage": "https://github.com/MetaMask/eslint-config#readme",
-  "devDependencies": {
-    "@typescript-eslint/eslint-plugin": "^4.14.2",
-    "@typescript-eslint/parser": "^4.14.2",
-    "eslint": "^7.7.0",
-    "eslint-plugin-import": "^2.22.0",
-    "eslint-plugin-jest": "^24.1.3",
-    "eslint-plugin-mocha": "^8.0.0",
-    "eslint-plugin-node": "^11.1.0",
-    "typescript": "^4.0.3"
-  },
-  "peerDependencies": {
-    "eslint": "^7.7.0",
-    "eslint-plugin-import": "^2.22.0"
-  }
+  "name": "root",
+  "private": true,
+  "workspaces": [
+    "packages/base"
+  ]
 }
```

Comment:
Should `lerna` be here as a dev dependency? :thinking:

Reply:
Hm, I think `lerna` recommends that you install it globally. I can either add it as a dev dep or add a note in the readme?

Reply:
Apparently not. From their README:

>Let's start by installing Lerna as a dev dependency of your project with npm.

Reply:
😅 56c43bc